### DISCUSSION TITLE
New CLI Undocumented

### DISCUSSION THREAD

**Discussion by letroot** [shadcn-ui/ui#4719] [General]
`npx shadcn init sidebar-01 login-01`

I'm confused as to how this new CLI command works. In the demo, it creates two pages, one with a login component and another with a sidebar.

Why does it work that way? Because I can't find anything in the shadcn docs talking about these two blocks: sidebar-01, login-01

**Comment by bouzidanas** [comment]
There is nothing yet in the docs as far as I can tell but there are some clues in the repository. There is [a registry of the blocks](https://github.com/shadcn-ui/ui/blob/078dfe66072c4ca780bbc99d4ad4b13b1f44fe7e/apps/www/registry/registry-blocks.ts#L25) that shows `login-01` and `sidebar-01` and you can see how they differ from other blocks listed.

For example, they same to be the only ones that have files of type `registry:page`.

It looks like for now, `login-01` and `sidebar-01` are the only options that will add some sort of page to the Next.js app.

**Reply by letroot** [reply]
Do you think it's going to get documented officially?
